packages/discord.js/package.json

```diff
@@ -12,7 +12,7 @@
     "prepublishOnly": "yarn lint && yarn test",
     "changelog": "git cliff --prepend ./CHANGELOG.md -l -c ./cliff.toml -r ../../ --include-path 'packages/discord.js/*'"
   },
-  "main": "./src/index.js",
+  "main": "./dist/src/index.js",
```

Comment:
Usually src is the source and dist is the compiled folder, shouldn’t it be just dist/index.js?

Reply:
We use the index.d.ts types which aren't in the source

Reply:
This could be the time to move that folder then since having dist/src doesn’t make much sense imo

Reply:
> This could be the time to move that folder then since having dist/src doesn’t make much sense imo

https://github.com/discordjs/discord.js/pull/7439#discussion_r803236250

Reply:
I don’t see how that’s related to what I said. I simply suggested moving the typings folder into src